package.json

```
@@ -73,7 +72,7 @@
     "should": "~13.2.3"
   },
   "engines": {
-    "node": ">=4.0.0"
+    "node": ">=10.0.0"
```

```suggestion
    "node": ">=10.12.0"
```

According to the [docs](https://nodejs.org/api/fs.html#fs_fs_mkdir_path_options_callback), this was only added in 10.12